src/animation/Animate.tsx

```diff
@@ -273,3 +273,16 @@ export class Animate extends PureComponent<AnimateProps, AnimateState> {
     return <div>{Children.map(children, child => cloneContainer(child))}</div>;
   }
 }
+
+export const AnimationManagerContext = createContext<AnimationManager | null>(null);
+
+export function Animate(props: AnimateProps) {
+  const contextAnimationManager = useContext(AnimationManagerContext);
```

Comment:
> 3. Add a utility that allows to replace animations manager on the fly so that we can do things like create a mock and then set animation progress in a unit tests without faffing around with requestAnimationFrame